provisioning/osc-install

```diff
@@ -145,7 +162,7 @@ setup_storage() {
 }
 
 setup_storage_on_all_hosts() {
-  for host in ${MASTER_HOSTNAME} ${NODE_HOSTNAMES//,/ }; do
+  for host in ${MASTER_HOSTNAME} ${NODE_HOSTNAMES//,/ } ${ETCD_HOSTNAMES//,/ }; do
```

Comment:
Should probably add functionality for etcd storage as otherwise we'll potentially be "wasting" too much storage as the etcd nodes don't need much.